CSharpSourceCode/Utilities/Extensions/HeroExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TaleWorlds.CampaignSystem;
+
+namespace TOW_Core.Utilities.Extensions
+{
+    public static class HeroExtensions
+    {
+        public static bool CanRaiseDead(this Hero hero)
+        {
+            return hero.IsHumanPlayerCharacter && hero.Culture.ToString().Equals("Vampire Counts");
```

Comment:
Please change this to a character attribute. Otherwise, nice job! no other comments from me